pkg/scalers/gcp_stackdriver_scaler.go

```diff
@@ -27,13 +23,13 @@ type stackdriverScaler struct {
 }
 
 type stackdriverMetadata struct {
-	projectID             string
-	filter                string
-	targetValue           float64
-	activationTargetValue float64
+	ProjectID             string  `keda:"name=projectID, order=triggerMetadata"`
+	Filter                string  `keda:"name=filter, order=triggerMetadata"`
+	TargetValue           float64 `keda:"name=targetValue, order=triggerMetadata, optional, default=5"`
+	ActivationTargetValue float64 `keda:"name=activationTargetValue, order=triggerMetadata, optional, default=0"`
```

Comment:
nit: `optional` is redundant with `default`
```suggestion
	TargetValue           float64 `keda:"name=targetValue, order=triggerMetadata, default=5"`
	ActivationTargetValue float64 `keda:"name=activationTargetValue, order=triggerMetadata, default=0"`
```

Reply:
Yes, thank you :)